v1/feed.yaml

```diff
@@ -0,0 +1,105 @@
+# Feed endpoints powered by the MCS
+
+swagger: 2.0
+paths:
+  /featured/{yyyy}/{mm}/{dd}:
+    get:
+      tags:
+        - Feed
+      summary: Aggregated featured content
+      description: |
+        Provides the aggregated feed content for the given date. The endpoint returns
+        the featured article of the day, most read articles for the previous day, news
+        content, a random article, and the featured image and video of the day.
+
+        Stability: [experimental](https://www.mediawiki.org/wiki/API_versioning#Experimental)
+      produces:
+        - application/json
+      parameters:
+        - name: yyyy
+          in: path
+          description: 'Year the aggregated content is requested for'
+          type: string
+          required: true
+        - name: mm
+          in: path
+          description: 'Month the aggregated content is requested for'
+          type: string
+          required: true
+        - name: dd
+          in: path
+          description: 'Day of the month the aggregated content is requested for'
+          type: string
+          required: true
+      responses:
+        '200':
+          description: JSON containing all of the featured content
+          schema:
+            type: object
+        default:
+          description: Error
+          schema:
+            $ref: '#/definitions/problem'
+      x-request-handler:
+        - from_mobileapps:
+            request:
+              method: get
+              uri: '{{options.host}}/{domain}/v1/feed/featured/{yyyy}/{mm}/{dd}'
```

Comment:
I think the aggregated endpoint will be the most requested feed endpoint. 'In the news' is likely updated most frequently (after random, of course).
So, for most of the endpoints pulling results from the aggregated endpoint makes sense to me.

The `random` component is an exception. Storage for this one doesn't seem necessary. IMO that'll probably just cause unnecessary churn for cassandra without much benefit. The 1-2 second caching should take most of the brunt anyways. If there are some clients getting different random results than others that's actually a good thing, and we wouldn't mind that at all.